.circleci/config.yml

```diff
@@ -5,14 +5,19 @@ orbs:
 
 jobs:
   build:
-    machine: true
+    machine:
+      image: circleci/classic:201808-01
     working_directory: ~/gusta-api
     steps:
       - checkout
-      - run: docker-compose up -d
+      - run: docker-compose up -d --build
       - run: docker-compose exec api npm run migrate
-      - run: docker-compose exec api npm test
-      - run: docker-compose exec api bash -c "bash <(curl -s https://codecov.io/env) && npx codecov"
```

Comment:
Turns out this doesn't work, so I ended up having to go a different way with it and run codecov outside the container.